app/graphql/types/customer_portal/customers/object.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Types
+  module CustomerPortal
+    module Customers
+      class Object < Types::BaseObject
+        graphql_name "CustomerPortalCustomer"
+
+        field :id, ID, null: false
+
+        field :currency, Types::CurrencyEnum, null: true
+        field :display_name, String, null: false
+        field :email, String, null: true
+        field :firstname, String
+        field :lastname, String
+        field :legal_name, String, null: true
+        field :legal_number, String, null: true
+        field :name, String
+        field :tax_identification_number, String, null: true
+
+        field :billing_configuration, Types::Customers::BillingConfiguration, null: true
+
+        # Billing address
+        field :address_line1, String, null: true
+        field :address_line2, String, null: true
+        field :city, String, null: true
+        field :country, Types::CountryCodeEnum, null: true
+        field :state, String, null: true
+        field :zipcode, String, null: true
+
+        field :shipping_address, Types::Customers::Address, null: true
+
+        def billing_configuration
+          {
+            id: "#{object&.id}-c0nf",
```

Comment:
why this?

Reply:
🤷‍♂️ I don't know, this is already in the admin site customers object type... not sure what does it means...

https://github.com/getlago/lago-api/blob/037290f94a6c25ce797cbad7c1af01c7aac0c64b/app/graphql/types/customers/object.rb#L135-L138

Reply:
@lovrocolic you introduced this change with the commit https://github.com/getlago/lago-api/commit/ce24c19fa267c0d14457042ddece5fafeced228e

Do you remember why?

Reply:
looks like frontend requires an id, for some reason it can't be random and we do not have a DB id as this information is defined within the customer record.

https://github.com/getlago/lago-api/pull/817#discussion_r1102645644